client/js/store/modules/auth.js

```diff
@@ -72,6 +72,7 @@ export default {
       return new Promise((resolve, reject) => {
         io.socket.get('/user/logout', {}, function handleResponse(resData, jwres) {
           if (jwres.statusCode === 200) {
+            context.commit('authFailure');
```

Comment:
Clears auth in store. Should probably rename the mutation

Reply:
I would definitely rename that sooner than later, it's a bit confusing.